Description: Build the library as shared
 Otherwise we are trying to link a static library built as PIE into a
 shared library.
Author: Guillem Jover <gjover@sipwise.com>

---

--- ngcp-sems-1.6.0.orig/apps/rtmp/librtmp/Makefile
+++ ngcp-sems-1.6.0/apps/rtmp/librtmp/Makefile
@@ -30,7 +30,7 @@ SO_posix=so.0
 SO_mingw=dll
 SO_EXT=$(SO_$(SYS))
 
-SHARED=no
+SHARED=yes
 SODEF_yes=-fPIC
 SOLIB_yes=librtmp.$(SO_EXT)
 SOINST_yes=install_$(SO_EXT)
